### PR TITLE
Add book Bash Scripting 

### PR DESCRIPTION
**Pull request by LeoDalcegio**
## What does this PR do?
Add resource

### Why is this valuable (or not)?
It is a great and free book for people who want to learn Bash Script

### How do we know it's really free?
It is on Creative Commons

### For book lists, is it a book? For course lists, is it a course? etc.
It is a book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -353,11 +353,12 @@
 * [O (comovente) guia de Ruby do Why](http://why.carlosbrando.com)
 
 
-### Shell
+### Shell / Bash
```
Comment (davorpa) [inline books/free-programming-books-pt_BR.md:356]: Please, give a reason for renaming

File: books/free-programming-books-pt_BR.md [hunk 2]
```diff
@@ -353,11 +353,12 @@
...
 
 * [Introdução ao Shell Script](http://aurelio.net/shell/apostila-introducao-shell.pdf) - Aurelio Marinho Jargas (PDF)
 * [Pequena introdução ao linux e ao Shell Script](https://www.telecom.uff.br/pet/petws/downloads/apostilas/LINUX.pdf) (PDF)
 * [Shell Script do Zero](http://metamorphoselinux.net/Shell_Script_do_Zero.pdf) (PDF)
+* [Bash Scripting](https://meleu.gitbooks.io/bashscripting/content/) - Meleu (gitbook)
```
Comment (davorpa) [inline books/free-programming-books-pt_BR.md:361]: Ensure linter alphabetization :wink:

**Review by davorpa — CHANGES_REQUESTED** [review]

File: books/free-programming-books-pt_BR.md (hunk 1 above)
Comment (LeoDalcegio) [reply books/free-programming-books-pt_BR.md:356]: I found the same on free-courses-fr.md and was trying to keep with the convention

File: books/free-programming-books-pt_BR.md (hunk 2 above)
Comment (LeoDalcegio) [reply books/free-programming-books-pt_BR.md]: Oh sorry, corrected it right now

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**